app/javascript/utilities/search/index.js

```diff
@@ -144,3 +144,14 @@ export function fetchSearch(endpoint, dataHash) {
     response.json(),
   );
 }
+
+/**
+ * A helper method to search for tags.
+ *
+ * @param {string} query - A partial or complete tag name that is being searched for.
+ *
+ * @returns {Promise<object>} A collection of found tags, if any.
+ */
+export function fetchTags(name) {
```

Comment:
I'm using this for this feature atm, but there are other places that can benefit from this helper function instead of using the generic `fetchSearch`. I may put this into a separate PR and update in the appropriate places.